src/OpenLayers/Controls/MousePosition.js

```diff
@@ -1274,6 +1274,9 @@ var MousePosition = (function (Control) {
             _timeout = 15000;
         }
 
+        // format de sortie si spécifié
+        var _outputFormat = options.outputFormat || "xml";
```

Comment:
Why not json by default? For lighter requests

Reply:
Agreed